src/defaults.mligo

```diff
@@ -14,6 +14,8 @@ let default_config (data : initial_config_data) : config = {
   voting_period = data.voting_period;
   max_proposals = 500n;
   max_votes = 1000n;
+  proposal_flush_time = data.proposal_flush_time;
+  proposal_expired_time = data.proposal_expired_time;
```

Comment:
To continue @sras 's [comment](https://github.com/tqtezos/baseDAO/pull/232#discussion_r632452443) here's where we should make the checks on those values.

Reply:
Nice, I added the check now.